server/middleware/authorization.js

```diff
@@ -0,0 +1,26 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config({path: "./vars/.env"});
+
+const authorization = (req,res,next)=>{
+
+    const jwtToken = req.header("token");
+
+    if(!jwtToken){
+        return res.status(403).json('Not Authorize');
+    }
+
+    try {
+
+        // verify current jwt token
+        const payload = jwt.verify(jwtToken, process.env.jwtSecret);
+        
+        req.user = payload.user;
+        next();
+    } catch (error) {
+        console.log(error.message);
```

Comment:
Why is the console.log here, I am not saying it should not be here, just asking :)

Reply:
For developer to see where is problem and which error appeared.

Reply:
This can stay for know but in production we use monitoring solutions such as CloudWatch and its impossible to filter out these errors.